src/NServiceBus.RabbitMQ/Configuration/ConnectionConfiguration.cs

```diff
@@ -31,7 +31,7 @@ class ConnectionConfiguration
 
         public string CertPassphrase { get; set; }
 
-        public IDictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
+        public Dictionary<string, object> ClientProperties { get; } = new Dictionary<string, object>();
```

Comment:
The reason for the change is that enumerating a `Dictionary` is allocation-free, but allocating `IDictionary` isn't.